### PR TITLE
Fix language selection not retained across some screens IDVA3-1293

### PR DESCRIPTION
**Pull request by hepsimo**
Screens: individual_statement, individualPscList, personal_code, psc_type
- add hidden input to propagate `lang` on screen template where missing
- add `"?lang="+lang` to button href on screen template -- to be replaced by future work

Other refactoring:
- add `test/` folder to file and folder naming rule checks
- refactor `/test` files/folders to conform to naming rules
- move eslint plugins to devDependencies section of `package.json`
- tidy file globs in "lint" scripts of `package.json`

### CODE REVIEW

File: src/views/router_views/psc_type/psc_type.njk [hunk 1]
```diff
@@ -35,6 +35,7 @@ summaryText: i18n.psc_type_details_title,
 text: i18n.psc_type_details_text
 }) }}
 
+<input type="hidden" name="lang" value="{{lang}}">
```
Comment (PhilipGlover) [inline src/views/router_views/psc_type/psc_type.njk:38]: I don't think this hidden input is needed at the moment.  On the Confirm Company screen there's a hidden input of `companyNumber` in order that the POST handling can access it again, from the request body.
But the `lang` is handled now in the request query.
I remove the `lang` input from the templates and the welsh journey seems to work.
However, when we do come to use forms maybe we'll need a input field for this.

**Review by PhilipGlover — APPROVED** [review]
- the welsh journey 'sticks' now all the way through to Confirmation and back to the Confirm Company (I know that errors, but that's because the company number is lost - that's separate.)
- all tests pass
- config changes look ok to me
- there's my comment re the hidden input `lang` but that doesn't stop me approving this.

**Review by hepsimo — COMMENTED** [review]
> I don't think this hidden input is needed at the moment.  On the Confirm Company screen there's a hidden input of `companyNumber` in order that the POST handling can access it again, from the request body.
But the `lang` is handled now in the request query.
I remove the `lang` input from the templates and the welsh journey seems to work.
However, when we do come to use forms maybe we'll need a input field for this. 

I'll check if this is still required.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**